src/main/java/com/saralein/server/response/FileHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.saralein.server.response;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class FileHelper {
+    String root;
+
+    public FileHelper(String root) {
+        this.root = root;
+    }
+
+    public String getRelativePath(String name) {
+        return "/" + root + "/" + name;
+    }
+
+    private File getDirectoryContents() {
+        return new File(root);
+    }
+
+    private File[] listDirectoryFiles() {
+        return getDirectoryContents().listFiles(
+            file -> !file.getName().equals(".DS_Store"));
+    }
```

Comment:
It might be easier if you returned a `List<File>` here. You can easily convert any array into a List by using `Arrays.asList`.

I tend to try to avoid using arrays directory, but that is just me 🤷‍♂️

Reply:
[Adjusted this](https://github.com/saralein/java-server/pull/2/commits/dba3c8c50f36e057acc529d833797c6300d578c1#diff-133e41c6e2f48487e5d6277fe3ce5cf1R24).  So nice that nothing else had to be adjusted for that change to work.  Also, I think I had an experience elsewhere with array vs. list where I can see why you'd prefer lists.  Will keep it in mind moving forward.